### PR TITLE
Adding Bagging chapter

### PR DESCRIPTION
**Pull request by AngelFelizR**

### CODE REVIEW

**Comment by AngelFelizR** [comment]
It seems I have problems with my description file

File: 10_bagging.Rmd [hunk 1]
```diff
@@ -1,13 +1,260 @@
...
+# Bagging (Bootstrap Aggregating)
 
-**Learning objectives:**
+It's a great way to the **reduce model variance** $\text{Var}(\hat{f}(x_0))$ and **improve accuracy** by reducing **overfitting**.
 
-- THESE ARE NICE TO HAVE BUT NOT ABSOLUTELY NECESSARY
+This method consist in:
 
-## SLIDE 1 {-}
+- Creating b bootstrap copies of the original training data
```
Comment (jonthegeek) [inline 10_bagging.Rmd]:
```suggestion
- Creating bootstrap copies of the original training data
```

**Review by jonthegeek — COMMENTED** [review]
I'll commit this typo fix and then approve!

**Review by jonthegeek — APPROVED** [review]
Thanks!

**Pull request merged**